### DISCUSSION TITLE
Can't link local database using CLI

### DISCUSSION THREAD

**Discussion by brettatoms** [supabase/supabase#4018] [Questions] [answered]
After removing all of my local `supabase/` containers and succesfully running `supabase init` I can't link my local database with my online database using the CLI.  Any help would be appreciated.

❯ supabase link --url 'postgresql://postgres:<password>@db.<project id>.supabase.co:5432/postgres'
Error: supabase_migrations.schema_migrations table conflicts with the contents of `migrations` directory.

**Comment by brettatoms** [comment, marked as answer]
It works if I create a new supabase project with an empty database.  I did have problems with my migrations in the previous project after I enabled the `pgaudit` extension so I'm not sure if its related to that.

**Reply by soedirgo** [reply]
I'm assuming you want to start the migrations from scratch, in which you can try deleting the `supabase_migrations` schema (I need to document this):
```sql
DROP SCHEMA supabase_migrations CASCADE;
```
Let me know if it works after that!

**Reply by brettatoms** [reply]
Sorry, I reset my environment from scratch to unblock me and I haven't run into the issue again so I can't test your suggestion.

**Reply by soedirgo** [reply]
No probs! Will mark the discussion as answered for now.

**Reply by brettatoms** [reply]
I did end up getting into this situation again and dropping the `supabase_migrations` fixed it.  Thanks for your help!

**Reply by eba8** [reply]
Where do you run `DROP SCHEMA supabase_migrations CASCADE;`? I'm in a somewhat similar situation.

**Reply by soedirgo** [reply]
@eba8 you'd run it on the SQL editor on the dashboard
